modules/nw-sriov-app-netutil.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/about-sriov.adoc
+
+[id="nw-sriov-app-netutil_{context}"]
+= DPDK library for use with container applications
+
+An link:https://github.com/openshift/app-netutil[optional library], `app-netutil`, provides several API methods for gathering network information about a pod from within a container running within that pod.
+
+This library is intended to assist with integrating SR-IOV virtual functions (VFs) in Data Plane Development Kit (DPDK) mode into the container.
+The library provides both a Golang API and a C API.
+
+Currently there are three API methods implemented:
+
+`GetCPUInfo()`:: This function determines which CPUs are available to the container and returns the list to the caller.
+
+`GetHugepages()`:: This function determines the amount of hugepage memory requested in the `Pod` spec for each container and returns the values to the caller.
++
+[NOTE]
+====
+Exposing hugepages via Kubernetes `Downward` API is an alpha feature in Kubernetes 1.20 and is not enabled in {product-title}. The API can be tested by enabling the feature gate, `FEATURE_GATES="DownwardAPIHugePages=true"`` on Kubernetes 1.20 or greater.
```

Comment:
`FEATURE_GATES="DownwardAPIHugePages=true"`` Double tick at the end. This was probably my fault, but if you are in there.

Reply:
@Billy99, oops that's probably me; I'll fix it.